documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: documentation
+menu: check
+subtitle: 'Tutorial: Using Check with SQL Server'
+---
+# Tutorial: Using Flyway Check with SQL Server
+
+{% include enterprise.html %}
+
+## Introduction
+
+This is an enteprise feature, but the beta version is available to Teams users.
+
+The new `check` command is run prior to migrating, to give us confidence in the state of the post-migration schema by producing a report. 
+
+One or more of the following flags must be set, which determine what the report contains:
+
+ - `-changes` produces a report of all the changes that will be applied to the schema in the next migration. 
+ - `-drift` produces a report showing objects in the schema which are not the result of any of the currently applied migrations, i.e. previous changes made outside of Flyway.
+
+More information can be found [here](/documentation/command/check).
+
+## Download the latest Beta version of Flyway Enterprise CLI
+
+First, [download the latest beta version of the Flyway Enterprise CLI](/documentation/usage/beta/) and extract its contents. 
+
+## Setting up databases
+
+To use `check` we will need access to two databases:
+
+- The **target** database. This is the database we're interested in making changes to. 
+- A **temporary** database. Flyway will clean this database, apply migrations to it, and make comparisons in order to generate **change and drift reports**. 
+
+In this example, there are two databases in our SQL Server instance: `foobar` (the target) and `check_temp_db`. Both can be accessed by the `sa` user.
+
+## Configuration
+
+As well as the usual config parameters (`flyway.url`, `flyway.user`, `flyway.password`, `flyway.licenseKey`...), we also need to configure properties specific to the `check` command (see [the documentation](/documentation/command/check) for more details). In this case, we only need to configure `flyway.check.tempUrl` as the other properties all have suitable default values.
+
+Our `flyway.conf` file (found in the `conf` folder), should contain the following values:
+
+```
+flyway.url=jdbc:sqlserver://localhost;databaseName=foobar;trustServerCertificate=true
+flyway.user=sa
+flyway.password=Flyway123
+flyway.licenseKey=<put your license key here>
+
+flyway.check.tempUrl=jdbc:sqlserver://localhost;databaseName=check_temp_db;trustServerCertificate=true
+```
+
+## Set up some migrations
+
+In the `sql` folder, we can create a new file `V1__add_table.sql`, containing a simple schema change:
+
+```sql
+create table PERSON (
+    ID int not null,
+    NAME varchar(100) not null
+);
+```
+## Detecting changes using `-changes` flag
```

Comment:
Potentially a daft question but if I had an empty DB and one pending migration as you imply - would check work or would it expect us to have something in the DB already ? Just wondering because if someone is playing around with this they might do just this

Reply:
It seems to work fine with an empty DB